src/components/starlight/EditLink.astro

```diff
@@ -1,4 +1,5 @@
 ---
+
```

Comment:
There's an extra space added here
```suggestion
```

Reply:
Yes, a mistake, but then I left it in in case anyone needed to see the EditLink logic easily. Can be removed now!  :laughing: